packages/sync/legacy/class.jetpack-sync-listener.php

```diff
@@ -158,7 +153,7 @@ function bulk_enqueue_full_sync_actions( $action_name, $args_array ) {
 			 *
 			 * @param array The action parameters
 			 */
-			$args = apply_filters( "jetpack_sync_before_enqueue_$action_name", $args );
+			$args        = apply_filters( "jetpack_sync_before_enqueue_$action_name", $args );
```

Comment:
```suggestion
			$args        = apply_filters(
				"jetpack_sync_before_enqueue_$action_name",
				$args
			);
```

Reply:
Thanks, but I disagree and prefer the original one.